src/fidesctl/core/config/logging_settings.py

```diff
@@ -0,0 +1,52 @@
+"""This module handles finding and parsing fides configuration files."""
+
+# pylint: disable=C0115,C0116, E0213
+import os
+from logging import DEBUG, INFO, getLevelName
+from typing import Union
+
+from pydantic import validator
+
+from .fides_settings import FidesSettings
+
+
+class FidesctlLoggingSettings(FidesSettings):
+    """Class used to store values from the 'logging' section of the config."""
+
+    # Logging
+    destination: str = "INFO"
+    level: Union[int, str] = ""
```

Comment:
Are these default values backwards?

```suggestion
    destination: str = ""
    level: Union[int, str] = "INFO"
```

Reply:
Thanks!